packages/esm-commons-lib/src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -188,14 +192,15 @@ export const EncounterList: React.FC<EncounterListProps> = ({
                 onClick={(e) => {
                   e.preventDefault();
                   launchEncounterForm(
-                    forms.find((form) => form.name == actionItem?.form?.name),
+                    formList[0],
```

Comment:
Why did u change this?  On certain scenarios, actions can have more than one form and that's why it was done this way. You could take a look at covid assessment

Reply:
I still don't get this, how does launching more than one form work? I thought the ohri form component only accepts one form uuid